Modules/RPC.cs

```diff
@@ -610,6 +611,131 @@ private static bool IsVersionMatch(int ClientId)
     }
 }
 
+[HarmonyPatch(typeof(PlayerPhysics), nameof(PlayerPhysics.HandleRpc))]
+internal class PlayerPhysicsRPCHandlerPatch
+{
+    private static bool hasVent(int ventId) => ShipStatus.Instance.AllVents.Any(v => v.Id == ventId);
+    private static bool hasLadder(int ladderId) => ShipStatus.Instance.Ladders.Any(l => l.Id == ladderId);
+
+    public static bool Prefix(PlayerPhysics __instance, byte callId, MessageReader reader)
+    {
+        var rpcType = (RpcCalls)callId;
+        MessageReader subReader = MessageReader.Get(reader);
```

Comment:
Better move EAC code to EAC.cs like this
![image](https://github.com/user-attachments/assets/12dff1f7-8fd1-4c87-a433-1005389e12c4)